_posts/2023-09-08-training-series.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "Learning SQL with Trino from the experts"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: 
+show_pagenav: false
+---
+
+Do you think SQL stands for Silly Quacking Leopards and write all your code in
+Java, Python, or Go? But you need to get data out of object storage in the cloud
+and some relational database at the same time? You should look at Trino and
+learning about SQL.
+
+Or do you know that ins and outs of joins, window functions, and your SQL
+queries are counted by the pages and not lines? You are the expert on SQL in
+your database team. You should also look at Trino and SQL.
+
+Luckily for you all we got the right SQL training coming up for everyone in our
+upcoming series with the founders of the Trino project and SQL experts Martin
+Traverso, Dain Sundstrom, and David Phillips, and myself as host and co-trainer.
+
+<!--more-->
+
+In the SQL training series we start with the basics around Trino, so you we are
```

Comment:
```suggestion
In the SQL training series, we start with the basics of Trino, so that we're
```

Reply:
I'd also consider saying "we'll start," as the future tense is fine here. We're making a promise, and it sounds more natural that way.

Reply:
Reworded all that some more.